crates/l1tx/src/filter/withdrawal_fulfillment.rs

```diff
@@ -110,6 +152,9 @@ mod test {
     use super::*;
     use crate::filter::types::{conv_deposit_to_fulfillment, OPERATOR_FEE};
 
+    // First 4 bytes of "strata"?  I am not sure what this is supposed to be.
+    const MAGIC: [u8; 4] = [b's', b't', b'r', b'a'];
```

Comment:
This is what I am planning to use in the bridge. Finishing the PR...
```suggestion
    // NOTE: magic bytes must be EXACTLY 4 bytes
    const MAGIC: [u8; 4] = b"alp\0";
```

Reply:
Wait, if we're hardcoding this value, why not just hard-code it as per the spec (i.e., `alpn`)? https://www.notion.so/SPS-50-L1-transaction-header-and-interpretation-1c5901ba000f80608cf8fdd9781666db?pvs=4

Reply:
We need to have a decision on this. I like `alpn` instead of `alp0` (all lowercase) but we need to make it consistent in `alpen` and `strata-bridge ` codebases.

Yay or nay for `alpn`?

Cc @barakshani

Reply:
we literally have months to bikeshed on this. let's merge this thing